app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -26,21 +35,51 @@ export const ListingDetailsAdditionalInformation = ({
             <TextTruncate text={listing.Listing_Other_Notes} />
```

Comment:
Is it possible to preserve the whitespace around these links? 
![image](https://user-images.githubusercontent.com/98352600/186545683-de2f0064-091b-449b-8bb1-5033ade87bd8.png)

Reply:
@christine-sfg Which listing is this?

Reply:
Sorry i should have included the listing info. [Here](https://dahlia-webapp-pr-1701.herokuapp.com/listings/a0W0P00000HbrAxUAJ?react=true) you go